Dockerfile

```diff
@@ -3,18 +3,25 @@
 
 ## Dev DOCKERFILE ##
 FROM alpine:3.20 as dev
+
 COPY bin/hcp /bin/hcp
+
 RUN apk --no-cache upgrade && apk --no-cache add \
 	bash \
-	curl \
 	jq \
-	nano \
-	vim
+	nano
+
+## No patch for vulnerabilities in alpine, downloading fixed versions from edge
+RUN apk --repository=https://dl-cdn.alpinelinux.org/alpine/edge/main --no-cache add \
+  vim=9.1.0678-r0 \
+  curl=8.9.1-r1
+
 RUN touch ~/.bashrc && hcp --autocomplete-install
+
 CMD ["/bin/bash"]
 
 ## DOCKERHUB DOCKERFILE ##
-FROM alpine:3.20 as release
+FROM dev as release
```

Comment:
Does this work? The build pipeline will try to build the docker target release: https://github.com/hashicorp/hcp/blob/main/.github/workflows/build.yml#L171.

The dev target, tries to copy bin/hcp to bin/hcp, but that may not exist when running the CI step. Is it safer just to duplicate the logic or to move to a script that both execute?

Reply:
Your right, thanks for pointing that out! Updated.